e2etests/features/support/common.rb

```diff
@@ -1,14 +1,3 @@
-# frozen_string_literal:true
-
-def user_access(user)
-  env = if user.downcase == "cms"
```

Comment:
This code will be needed for CMS

Reply:
We can add it back when the CMS pages have tests

Until then, it is confusing